.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,21 @@
-<!-- To make our job easier, please spend time to review your application before submitting. -->
-<!-- To tick a box, place an `x` between two square brackets e.g. [x] -->
+## is-a.dev Subdomain Registration
+Please answer below questions.
 
-## Requirements
-Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements below.
-Please note that we reserve the rights not to accept any domain at our own discretion.
+### How do you plan to use your subdomain?
 
+### What is the purpose of these records?
+→
+
+### Requirements
 - [ ] The file is in the `domains` folder and is in the JSON format.
 - [ ] You have completed your website. <!-- This is not required if the domain you're registering is for emails. -->
 - [ ] The website is reachable.  <!-- This is not required if the domain you're registering is for emails. -->
 - [ ] You're not using Vercel or Netlify.  <!-- This is not required if you're using an URL record. -->
 - [ ] The CNAME record doesn't contain `https://` or `/`.  <!-- This is not required if you are not using a CNAME record. -->
 - [ ] There is sufficient information at the `owner` field.  <!-- You need to have your email presented at `email` field. If you don't want to provide your email for any reason, you can specify another social platform (e.g. Discord or Twitter) so we can contact you. -->
 
-## Link to Website
-<!-- Please provide a link to your website below. -->
+* Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.
```

Comment:
```suggestion
- Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.
```